### ISSUE TITLE
New message notification (in Fedora36/Gnomr) doesn't take you to the app when clicked

### ISSUE THREAD

**Issue by userjlegwork** [telegramdesktop/tdesktop#25216]
### Steps to reproduce

1. Open Telgram on Fedora 36 Gnome edition
2. Move to another app/virtual desktop (just lose Telgram focus)
3. Trigger a notification
4. Click on the notification (Nothing happens)

### Expected behaviour

Telegram should open on the notification's message 

### Actual behaviour

Focus stays the same, when you look for Telegram the message from the notification is open but didn't auto redirect to the app (focus) when notification was clicked

### Operating system

Fedora Workstation 36

### Version of Telegram Desktop

4.2.4

### Installation source

Static binary from official website

**Comment by ilya-fedin** [comment]
Are you using Wayland? If yes, are you sure GNOME supports [xdg-activation with notifications](https://gitlab.freedesktop.org/xdg/xdg-specs/-/merge_requests/43)? Without this, tdesktop won't be able to focus and there's nothing could be done about that.

**Comment by userjlegwork** [comment]
> Are you using Wayland? If yes, are you sure GNOME supports [xdg-activation with notifications](https://gitlab.freedesktop.org/xdg/xdg-specs/-/merge_requests/43)? Without this, tdesktop won't be able to focus and there's nothing could be done about that.

Yes, I'm using Wayland. I think Gnome does use the standard xdg activation with notifications, maybe it could be related with [this](https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/4535)?

**Comment by ilya-fedin** [comment]
> maybe it could be related with [this](https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/4535)

tdesktop doesn't use gtk, so any changes to gtk don't matter

> I think Gnome does use the standard xdg activation with notifications

The fact it doesn't work for you means that they don't support it most likely.
